### ISSUE TITLE
As an Open Forest User I'd like to know that the site is 508 Compliant so that I can use alternative methods to navigate the site if needbe

### ISSUE THREAD

**Issue by Dmac26** [USDAForestService/fs-open-forest-platform#1478]
## Notes
Per USDA standards all sites should be 508 Accessibility compliant to ensure that those with a need to navigate the site through alternative methods may do so. This means that the site should be navigable through text-to-speech features as well as being accessible and easily view-able for those with color-blindness. To achieve this end 508 testing was performed on the Open Forest platform through both automated and manual means. This card will be complete when a sufficient number problems found through 508 testing have been re-mediated and and re-tested.

Summaries of the testing done on both the Staging and Production site can be found below:

## Tasks
- [ ] Ensure markup documents contain well-formed elements
- [ ] Ensure ARIA roles, states, and properties are valid
- [ ] Ensure new windows or frames do not open without a warning
- [ ] Ensure embedded elements and canvas elements provide a meaningful text equivalent
- [ ] Provide a valid label for form fields
- [ ] Complete 508 automated and manual testing again after addressing the above
- [ ] Fix any new or persistent issues
- [ ] PO approved

## Definition of Done
- [ ] 508 testing produces a passing score
- [ ] Pull requests meet technical definition of done
- [ ] Usability tested

**Comment by carlsonem** [comment]
@Dmac26 where does this stand? Are you ready for validation?

**Comment by Dmac26** [comment]
@Rebekah-Hernandez - Would you please independently verify that this is passing 508 compliance on AMP?
